### ISSUE TITLE
Pre-work Checklist: Developer: Brian Mui

### ISSUE THREAD

**Issue by bootcamp-brian** [hackforla/website#3912]
### Prerequisite
We are looking forward to having you on our team. Please make sure to attend the general Hack for LA onboarding to get the process started https://meetup.com/hackforla/events.

### Overview
As a new developer on the HfLA website team, fill in the following fields as you complete each onboarding item.

### Special Notes
1. It may take you a few weeks to finish this issue, because part of it is learning how to provide updates on issues that take more than a week to complete.    Please keep this issue open until you have been able to do all the steps.  
2. Usually we don't want to you have more than one issue assigned to you at a time, this issue is the exception, because it is instructions on how to work on other issues.
3. The action items listed below should mostly be worked on in a sequential order. However, you don't have to wait on one if you can proceed with the others, For instance, you don't have to wait for attending a weekly meeting before setting up your dev environment.  

### Action Items

- [x] Add yourself to the #hfla-site and #hfla-site-pr Slack channels
- [x] Share your GitHub handle and Gmail address in the hfla-site-onboarding slack channel so you can be added to the Google Drive (alternative to go to meeting and direct message a merge team member at the [meeting](https://github.com/hackforla/website/wiki/Meetings-and-Agendas
))
  - [x] To find contact information for the merge team members and technical leads, please take a look at our [Meet the Team wiki page](https://github.com/hackforla/website/wiki/Meet-the-Team)
- [x] Also, confirm with a merge team member or a technical lead that they have added you to the Google Calendar invites for our Zoom meetings
- [x] (Once added to the Drive) Add yourself to the [team roster](https://docs.google.com/spreadsheets/d/11u71eT-rZTKvVP8Yj_1rKxf2V45GCaFz4AXA7tS_asM/edit#gid=0)
- [x] After you have finished adding yourself to the roster, let a merge team member or a technical lead know you have added yourself to the roster and would like to be added to the `website-write` and `website` teams on GitHub
- [x] Once added to the website-write team:
  - [x] Self Assign this issue (gear in right side panel)
  - [x] Add the "role: front end" or "role: back end" or both label(s) to this issue and remove the "role missing" label (gear in right side panel)
  - [x] Add this issue to the Project Board under the Projects section (gear in right side panel)
- [x] Signup for a [Figma](https://Figma.com) account
- [x] Attend weekly team meetings:
  - [x] Developer (front-end/back-end) weekly team meeting, Tuesdays 7-8pm PST
  - [ ] (Optional) Office Hours, Thursdays 7-8pm PST
  - [x] All team meeting (UX, Development, Product), Sunday 10am-12pm PST
- [x] Note: There are no meetings on the 1st-7th of every month.
- [x] Note regarding weekly team meeting requirements: All website team members are required to attend at least 1 team meeting in a week (held on Tuesdays, Thursdays and Sundays). In case, you are unable in any given week, you should reach out to the tech leadership team. Exceptions to this requirement may be provided on a case-by-case basis. Also, please let the tech leadership team know (through a slack message in the hfla-site channel as well as an @ mention in a comment of the issue that you would be working on) if you are planning to take a week off or a longer vacation.
- [x] Complete steps 1.1 - 1.6 in [Part 1: Setting up the development environment within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-1-setting-up-the-development-environment)
  - [x] OPTIONAL: If you run into any issues, use [4.1 How do I ask for help within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#41-what-do-i-do-if-i-need-help) as a visual example to get a hold of our [HfLA website leadership team](https://github.com/hackforla/website/projects/7#card-69730135). That way we can help you resolve any set up issues immediately.
- [x] Read section 2.1 - 2.4 in [Part 2: How the Website team works with GitHub issues within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-2-how-the-website-team-works-with-github-issues)
  - [x] Follow the steps in section [2.3 Where can I find GitHub issues to work on?](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#23-where-can-i-find-github-issues-to-work-on) to assign yourself your first issue under the Project Board's Prioritized Backlog column and use [2.7 Working on a Issue within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27-working-on-an-issue) to start working on your issue
  - [x] Read [2.6 What to do when you need to stop mid issue](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#26-what-to-do-when-you-need-to-stop-mid-issue)
- [x] Once you take a good first issue, you will provide estimates and progress reports. Also, once you finish providing your "Estimate" action item below, please answer the question in the "Time spent so far" action item (also below).
   - [x] Estimate: Copy the below and put it in the "good first issue" that you picked.
 Check this box when you have completed this task so that we can identify if you understood the instructions and know what to do on all subsequent issues upon assignment. (Note: Please provide estimates on all the issues that you pick up, going forward.)
    ```
   Availability for this week:
   My estimated ETA for completing this issue:
    ```
   - [x] Time spent so far: Copy the question below into a new comment below (in this pre-work issue only) and answer it. This is just to get feedback on how long it took you to get to this point on this pre-work. There is no right or wrong answers. There is no judgement. It is ok if you take a long time or if you do it really fast or at any pace.
    ```
    How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
    ```
   - [x] Progress Reports: Copy the below and put it in the issue once you have been assigned to the issue at least 5 days (we check weekly on Fridays), or sooner if you have something to report.  If you finish this issue before 5 days are reached, Yeah!!, do it on your next issue. **This update should be done every week for every issue that you are assigned to**. The checkbox here is meant for us to see if you understood the instructions when you end up doing your first weekly progress update.
    ```
    Provide Update
    1. Progress
    2. Blockers
    3. Availability
    4. ETA
    ```
- [x] Read sections 3.1.a - 3.1.c in [3.1 How to make a pull request](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#31-how-to-make-a-pull-request) to learn more about how to make a pull request (PR) for the issue that you are working on and how to make changes to your PR if changes are requested by the reviewer
- [x] Additional notes:
   - [x] Please don't forget to review pull requests whenever you are able to.  The [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests) guide will teach you how to review pull requests.
   - [x] Please work on only one issue at a time and wait until your pull request is merged before picking up another issue.
- [x] Read and understand how we progress through issues. Then, you can check this off.
Progress through issues with increasing complexity in the following order:
  - Good first issue (one per person)
  - Good second issue (one per person)
  - Small (one per person, with some exceptions, see below)
  - Medium (you can work on more than one medium issue, but only one at a time)
  - Large (you can work on more than one large issue, but only one at a time)
    - The reasons for this progression are:
      - The issues start out as being prescriptive and become less so as you gain more experience by working through increasingly complex issues.
      - We are trying to teach you the team methodology through the issues themselves.
      - It ensures you understand what we expect and the quality of contributions.
    - You can work on back-to-back small issues if it meets the following criteria:
      - You are learning something new and need to work on an issue of a lesser complexity
      - Special request by a lead or pm
- [x] Read the [Start Here - Developers](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=8583%3A0) in Figma
- [x] Go familiarize yourself with the [Hack for LA Design System page in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)  (where you can see components and their SCSS classes)
- [x] Once all tasks are completed, close this issue.  This issue will remain open until you get through your first progress report.  Once you have done that, we are confident you know how to keep the momentum going on your issue and keep your team informed.

### What should I do if I have a question about an issue I'm working on, and I haven't gotten a response yet?
- First, you should post the question or blocker as a comment on your assigned issue, so it can be easily referred to in the next bullet points.
- Then, add the label "Status: Help Wanted" so other developers can see it and potentially help answer your question. In addition, you will still need to post a Slack message or bring it up in meeting so we know you need help; see below for how to do that.
- Also, you can post your question on the hfla-site slack channel and link the issue you're working on, so other developers can see and respond.
- Lastly, you can add the issue to the "Development team meeting discussion items" column of the Project Board so that it can be addressed in the next development meeting. Please bring it during the meeting that you need help.

### Resources/Instructions
- [Contributing.md - Hack for LA](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
- [GitHub Project Board - Hack for LA](https://github.com/hackforla/website/projects/7)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Front End](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Back End/DevOps](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22)
- [Figma - Hack for LA](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website)
- [Google Drive - Hack for LA website team](https://drive.google.com/drive/folders/1p76K0FgfiAWeIIEyoyJ_Iik8FVj8cBjT?usp=sharing)
- [Agenda / Notes - Dev Team Tuesday meeting](https://github.com/hackforla/website/issues/2010)
- [Agenda / Notes - All Team meeting](https://github.com/hackforla/website/issues/2027)
- [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests)

**Comment by bootcamp-brian** [comment]
How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?

I wasn't paying close attention to the time spent but my estimate is 3-4 hours. I haven't attended a weekly meeting yet since I was told that the team I'm on doesn't meet during the first week of the month.

**Comment by bootcamp-brian** [comment]
Didn't realize I needed to post a progress update on the pre-work checklist

Progress: Just have finishing a large issue left
Blockers: None yet
Availability: 6-10hrs
ETA: 2 weeks

**Comment by ExperimentsInHonesty** [comment]
Has provided two progress updates in this issue https://github.com/hackforla/website/issues/2651#issuecomment-1464824468 

This prework has been successfully completed!

**Comment by HackforLABot** [comment]
## Activity Log: bootcamp-brian
### Repo: https://github.com/hackforla/website

#####  ⚠ Important note: The bot updates this comment automatically - do not edit

- bootcamp-brian opened: https://github.com/hackforla/website/issues/3912 at 01/31/2023, 08:58 PM PST
- bootcamp-brian assigned: https://github.com/hackforla/website/issues/3912 at 01/31/2023, 08:58 PM PST
- bootcamp-brian assigned: https://github.com/hackforla/website/issues/3854 at 02/05/2023, 01:36 PM PST
- bootcamp-brian closed by PR 4273: https://github.com/hackforla/website/issues/2833#event-8441846519 at 02/05/2023, 02:18 PM PST
- bootcamp-brian reopened: https://github.com/hackforla/website/issues/2833#event-8441846519 at 02/05/2023, 02:18 PM PST
- bootcamp-brian commented on: https://github.com/hackforla/website/issues/3854#issuecomment-1418280947 at 02/05/2023, 02:21 PM PST
- bootcamp-brian commented on: https://github.com/hackforla/website/issues/3912#issuecomment-1418281696 at 02/05/2023, 02:25 PM PST
- bootcamp-brian opened: https://github.com/hackforla/website/pull/3940 at 02/05/2023, 02:44 PM PST
- bootcamp-brian commented on: https://github.com/hackforla/website/issues/3854#issuecomment-1418286311 at 02/05/2023, 02:49 PM PST
- bootcamp-brian issue closed by PR 3940: https://github.com/hackforla/website/issues/3854#event-8444409867 at 02/05/2023, 11:33 PM PST
- bootcamp-brian PR was merged: https://github.com/hackforla/website/pull/3940#event-8464545142 at 02/07/2023, 08:37 PM PST
- bootcamp-brian issue closed by PR 3940: https://github.com/hackforla/website/issues/3854#event-8464545390 at 02/07/2023, 08:37 PM PST
- bootcamp-brian commented on: https://github.com/hackforla/website/pull/3965#issuecomment-1428563218 at 02/13/2023, 11:51 AM PST
- bootcamp-brian submitted review: https://github.com/hackforla/website/pull/3965#pullrequestreview-1296355586 at 02/13/2023, 12:00 PM PST
- bootcamp-brian commented on: https://github.com/hackforla/website/pull/3965#issuecomment-1428625729 at 02/13/2023, 12:20 PM PST
- bootcamp-brian submitted review: https://github.com/hackforla/website/pull/3965#pullrequestreview-1296503294 at 02/13/2023, 01:55 PM PST
- bootcamp-brian submitted review: https://github.com/hackforla/website/pull/3965#pullrequestreview-1296536656 at 02/13/2023, 02:25 PM PST
- bootcamp-brian commented on: https://github.com/hackforla/website/pull/3965#issuecomment-1428932849 at 02/13/2023, 04:53 PM PST
- bootcamp-brian submitted review: https://github.com/hackforla/website/pull/3965#pullrequestreview-1296701990 at 02/13/2023, 05:34 PM PST
- bootcamp-brian assigned: https://github.com/hackforla/website/issues/3958#issuecomment-1427172910 at 02/14/2023, 08:22 PM PST
- bootcamp-brian commented on: https://github.com/hackforla/website/issues/3958#issuecomment-1430739814 at 02/14/2023, 08:25 PM PST
- bootcamp-brian opened: https://github.com/hackforla/website/pull/3982 at 02/14/2023, 09:15 PM PST
- bootcamp-brian commented on: https://github.com/hackforla/website/issues/3958#issuecomment-1430772034 at 02/14/2023, 09:19 PM PST
- bootcamp-brian PR was merged: https://github.com/hackforla/website/pull/3982#event-8534502136 at 02/16/2023, 12:55 AM PST
- bootcamp-brian issue closed by PR 3982: https://github.com/hackforla/website/issues/3958#event-8534502397 at 02/16/2023, 12:55 AM PST
- bootcamp-brian commented on: https://github.com/hackforla/website/pull/3988#issuecomment-1433627095 at 02/16/2023, 11:50 AM PST
- bootcamp-brian submitted review: https://github.com/hackforla/website/pull/3988#pullrequestreview-1302214034 at 02/16/2023, 12:15 PM PST
- bootcamp-brian commented on: https://github.com/hackforla/website/pull/3980#issuecomment-1433683848 at 02/16/2023, 12:36 PM PST
- bootcamp-brian submitted review: https://github.com/hackforla/website/pull/3980#pullrequestreview-1302262372 at 02/16/2023, 12:53 PM PST
- bootcamp-brian assigned: https://github.com/hackforla/website/issues/4009 at 02/23/2023, 09:46 AM PST
- bootcamp-brian commented on: https://github.com/hackforla/website/issues/4009#issuecomment-1442188021 at 02/23/2023, 09:47 AM PST
- bootcamp-brian opened: https://github.com/hackforla/website/pull/4032 at 02/23/2023, 10:20 AM PST
- bootcamp-brian commented on: https://github.com/hackforla/website/issues/4009#issuecomment-1442232342 at 02/23/2023, 10:23 AM PST
- bootcamp-brian commented on: https://github.com/hackforla/website/pull/4030#issuecomment-1442324879 at 02/23/2023, 11:37 AM PST
- bootcamp-brian submitted review: https://github.com/hackforla/website/pull/4030#pullrequestreview-1312027531 at 02/23/2023, 11:43 AM PST
- bootcamp-brian commented on: https://github.com/hackforla/website/pull/4021#issuecomment-1442356284 at 02/23/2023, 11:58 AM PST
- bootcamp-brian submitted review: https://github.com/hackforla/website/pull/4021#pullrequestreview-1312063383 at 02/23/2023, 12:06 PM PST
- bootcamp-brian PR was merged: https://github.com/hackforla/website/pull/4032#event-8598478815 at 02/23/2023, 10:03 PM PST
- bootcamp-brian issue closed by PR 4032: https://github.com/hackforla/website/issues/4009#event-8598478958 at 02/23/2023, 10:03 PM PST
- bootcamp-brian commented on: https://github.com/hackforla/website/pull/4032#issuecomment-1442877491 at 02/23/2023, 10:46 PM PST
- bootcamp-brian assigned: https://github.com/hackforla/website/issues/4028 at 02/25/2023, 07:46 PM PST
- bootcamp-brian commented on: https://github.com/hackforla/website/issues/4028#issuecomment-1445261218 at 02/25/2023, 07:47 PM PST
- bootcamp-brian commented on: https://github.com/hackforla/website/issues/4028#issuecomment-1445267738 at 02/25/2023, 08:44 PM PST
- bootcamp-brian opened: https://github.com/hackforla/website/pull/4052 at 02/26/2023, 02:12 PM PST
- bootcamp-brian commented on: https://github.com/hackforla/website/issues/4028#issuecomment-1445481806 at 02/26/2023, 02:14 PM PST
- bootcamp-brian PR was merged: https://github.com/hackforla/website/pull/4052#event-8617362886 at 02/27/2023, 07:56 AM PST
- bootcamp-brian issue closed by PR 4052: https://github.com/hackforla/website/issues/4028#event-8617363237 at 02/27/2023, 07:56 AM PST
- bootcamp-brian commented on: https://github.com/hackforla/website/pull/4052#issuecomment-1446844479 at 02/27/2023, 10:28 AM PST
- bootcamp-brian commented on: https://github.com/hackforla/website/pull/4092#issuecomment-1452832233 at 03/02/2023, 06:02 PM PST
- bootcamp-brian submitted review: https://github.com/hackforla/website/pull/4092#pullrequestreview-1323008304 at 03/02/2023, 06:14 PM PST
- bootcamp-brian assigned: https://github.com/hackforla/website/issues/2651#issuecomment-1003694534 at 03/04/2023, 11:26 AM PST
- bootcamp-brian commented on: https://github.com/hackforla/website/issues/2651#issuecomment-1454850031 at 03/04/2023, 11:28 AM PST
- bootcamp-brian commented on: https://github.com/hackforla/website/issues/3912#issuecomment-1454850560 at 03/04/2023, 11:30 AM PST
- bootcamp-brian commented on: https://github.com/hackforla/website/pull/4139#issuecomment-1462700821 at 03/09/2023, 12:07 PM PST
- bootcamp-brian submitted review: https://github.com/hackforla/website/pull/4139#pullrequestreview-1333726390 at 03/09/2023, 12:16 PM PST
- bootcamp-brian commented on: https://github.com/hackforla/website/pull/4140#issuecomment-1462713091 at 03/09/2023, 12:19 PM PST
- bootcamp-brian submitted review: https://github.com/hackforla/website/pull/4140#pullrequestreview-1333740408 at 03/09/2023, 12:22 PM PST
- bootcamp-brian commented on: https://github.com/hackforla/website/pull/4119#issuecomment-1462723917 at 03/09/2023, 12:24 PM PST
- bootcamp-brian submitted review: https://github.com/hackforla/website/pull/4119#pullrequestreview-1333749348 at 03/09/2023, 12:26 PM PST
- bootcamp-brian commented on: https://github.com/hackforla/website/issues/2651#issuecomment-1464824468 at 03/10/2023, 08:43 PM PST
- bootcamp-brian commented on: https://github.com/hackforla/website/pull/4196#issuecomment-1472854314 at 03/16/2023, 03:37 PM PDT
- bootcamp-brian submitted review: https://github.com/hackforla/website/pull/4196#pullrequestreview-1344874138 at 03/16/2023, 03:46 PM PDT
- bootcamp-brian commented on: https://github.com/hackforla/website/pull/4195#issuecomment-1472869372 at 03/16/2023, 03:57 PM PDT
- bootcamp-brian submitted review: https://github.com/hackforla/website/pull/4195#pullrequestreview-1344908839 at 03/16/2023, 04:05 PM PDT
- bootcamp-brian commented on: https://github.com/hackforla/website/issues/2651#issuecomment-1474387715 at 03/17/2023, 01:46 PM PDT
- bootcamp-brian commented on: https://github.com/hackforla/website/pull/4205#issuecomment-1475333742 at 03/19/2023, 10:31 AM PDT
- bootcamp-brian commented on: https://github.com/hackforla/website/pull/4203#issuecomment-1475333792 at 03/19/2023, 10:32 AM PDT
- bootcamp-brian submitted review: https://github.com/hackforla/website/pull/4205#pullrequestreview-1347569017 at 03/19/2023, 12:58 PM PDT
- bootcamp-brian submitted review: https://github.com/hackforla/website/pull/4203#pullrequestreview-1347569819 at 03/19/2023, 01:04 PM PDT
- bootcamp-brian submitted review: https://github.com/hackforla/website/pull/4205#pullrequestreview-1347572720 at 03/19/2023, 01:26 PM PDT
- bootcamp-brian commented on: https://github.com/hackforla/website/pull/4260#issuecomment-1481814809 at 03/23/2023, 01:00 PM PDT
- bootcamp-brian submitted review: https://github.com/hackforla/website/pull/4260#pullrequestreview-1355495194 at 03/23/2023, 01:09 PM PDT
- bootcamp-brian opened: https://github.com/hackforla/website/issues/4264 at 03/23/2023, 01:49 PM PDT
- bootcamp-brian closed as completed: https://github.com/hackforla/website/issues/3912#event-8841847133 at 03/24/2023, 11:52 AM PDT
- bootcamp-brian commented on: https://github.com/hackforla/website/issues/2651#issuecomment-1483732033 at 03/24/2023, 10:40 PM PDT
- bootcamp-brian commented on: https://github.com/hackforla/website/pull/4290#issuecomment-1484163266 at 03/26/2023, 10:20 AM PDT
- bootcamp-brian commented on: https://github.com/hackforla/website/pull/4278#issuecomment-1484163296 at 03/26/2023, 10:20 AM PDT
- bootcamp-brian submitted review: https://github.com/hackforla/website/pull/4278#pullrequestreview-1359793222 at 03/27/2023, 01:05 PM PDT
- bootcamp-brian submitted review: https://github.com/hackforla/website/pull/4290#pullrequestreview-1359813967 at 03/27/2023, 01:21 PM PDT
- bootcamp-brian submitted review: https://github.com/hackforla/website/pull/4290#pullrequestreview-1361812391 at 03/28/2023, 12:54 PM PDT
- bootcamp-brian submitted review: https://github.com/hackforla/website/pull/4260#pullrequestreview-1361815745 at 03/28/2023, 12:56 PM PDT
- bootcamp-brian commented on: https://github.com/hackforla/website/pull/4278#issuecomment-1487521436 at 03/28/2023, 01:02 PM PDT
- bootcamp-brian submitted review: https://github.com/hackforla/website/pull/4278#pullrequestreview-1363698609 at 03/29/2023, 11:11 AM PDT
- bootcamp-brian issue closed as completed: https://github.com/hackforla/website/issues/2651#event-8893197675 at 03/30/2023, 06:32 PM PDT
- bootcamp-brian commented on: https://github.com/hackforla/website/pull/4428#issuecomment-1499517049 at 04/06/2023, 12:26 PM PDT
- bootcamp-brian submitted review: https://github.com/hackforla/website/pull/4428#pullrequestreview-1375449391 at 04/06/2023, 12:28 PM PDT
- bootcamp-brian commented on: https://github.com/hackforla/website/pull/4430#issuecomment-1499525917 at 04/06/2023, 12:36 PM PDT
- bootcamp-brian submitted review: https://github.com/hackforla/website/pull/4430#pullrequestreview-1375460702 at 04/06/2023, 12:39 PM PDT
- bootcamp-brian commented on: https://github.com/hackforla/website/pull/4433#issuecomment-1499532482 at 04/06/2023, 12:43 PM PDT
- bootcamp-brian submitted review: https://github.com/hackforla/website/pull/4433#pullrequestreview-1375468548 at 04/06/2023, 12:46 PM PDT
- bootcamp-brian assigned: https://github.com/hackforla/website/issues/4249 at 04/06/2023, 12:48 PM PDT
- bootcamp-brian commented on: https://github.com/hackforla/website/issues/4249#issuecomment-1499540164 at 04/06/2023, 12:51 PM PDT
- bootcamp-brian commented on: https://github.com/hackforla/website/pull/4433#issuecomment-1499548400 at 04/06/2023, 01:00 PM PDT
- bootcamp-brian submitted review: https://github.com/hackforla/website/pull/4433#pullrequestreview-1375531568 at 04/06/2023, 01:28 PM PDT
- bootcamp-brian opened: https://github.com/hackforla/website/issues/4479 at 04/11/2023, 08:01 PM PDT
- bootcamp-brian commented on: https://github.com/hackforla/website/issues/4249#issuecomment-1509479700 at 04/14/2023, 07:58 PM PDT
- bootcamp-brian commented on: https://github.com/hackforla/website/pull/4545#issuecomment-1514077867 at 04/18/2023, 08:21 PM PDT
- bootcamp-brian commented on: https://github.com/hackforla/website/issues/4249#issuecomment-1514091396 at 04/18/2023, 08:46 PM PDT
- bootcamp-brian submitted review: https://github.com/hackforla/website/pull/4545#pullrequestreview-1396521415 at 04/21/2023, 07:13 PM PDT
- bootcamp-brian commented on: https://github.com/hackforla/website/issues/4249#issuecomment-1518536843 at 04/21/2023, 11:53 PM PDT
- bootcamp-brian commented on: https://github.com/hackforla/website/pull/4496#issuecomment-1519131586 at 04/23/2023, 11:53 AM PDT
- bootcamp-brian opened: https://github.com/hackforla/website/issues/4568 at 04/26/2023, 04:42 PM PDT
- bootcamp-brian commented on: https://github.com/hackforla/website/pull/4545#issuecomment-1524234312 at 04/26/2023, 04:58 PM PDT
- bootcamp-brian commented on: https://github.com/hackforla/website/pull/4545#issuecomment-1524258489 at 04/26/2023, 05:11 PM PDT
- bootcamp-brian merged: https://github.com/hackforla/website/pull/4570#event-9119036119 at 04/27/2023, 02:45 PM PDT
- bootcamp-brian closed by PR 4570: https://github.com/hackforla/website/issues/4413#event-9119036247 at 04/27/2023, 02:45 PM PDT
- bootcamp-brian commented on: https://github.com/hackforla/website/pull/4545#issuecomment-1526557605 at 04/27/2023, 02:50 PM PDT
- bootcamp-brian submitted review: https://github.com/hackforla/website/pull/4545#pullrequestreview-1404953673 at 04/27/2023, 02:50 PM PDT
- bootcamp-brian submitted review: https://github.com/hackforla/website/pull/4545#pullrequestreview-1405096816 at 04/27/2023, 06:01 PM PDT
- bootcamp-brian commented on: https://github.com/hackforla/website/pull/4545#issuecomment-1526850935 at 04/27/2023, 06:26 PM PDT
- bootcamp-brian commented on: https://github.com/hackforla/website/issues/4568#issuecomment-1526855180 at 04/27/2023, 06:32 PM PDT
- bootcamp-brian commented on: https://github.com/hackforla/website/pull/4545#issuecomment-1526906517 at 04/27/2023, 07:50 PM PDT
- bootcamp-brian submitted review: https://github.com/hackforla/website/pull/4545#pullrequestreview-1406612199 at 04/28/2023, 02:14 PM PDT
- bootcamp-brian merged: https://github.com/hackforla/website/pull/4545#event-9128474995 at 04/28/2023, 02:15 PM PDT
- bootcamp-brian closed by PR 4545: https://github.com/hackforla/website/issues/4158#event-9128475122 at 04/28/2023, 02:15 PM PDT
- bootcamp-brian commented on: https://github.com/hackforla/website/issues/4249#issuecomment-1528903708 at 04/29/2023, 05:19 PM PDT
- bootcamp-brian commented on: https://github.com/hackforla/website/pull/4496#issuecomment-1532197253 at 05/02/2023, 02:47 PM PDT
- bootcamp-brian opened: https://github.com/hackforla/website/pull/4600 at 05/04/2023, 05:49 PM PDT
- bootcamp-brian assigned: https://github.com/hackforla/website/issues/4568 at 05/07/2023, 09:24 AM PDT
- bootcamp-brian assigned: https://github.com/hackforla/website/issues/4605 at 05/11/2023, 11:12 AM PDT
- bootcamp-brian commented on: https://github.com/hackforla/website/issues/4605#issuecomment-1544464036 at 05/11/2023, 11:13 AM PDT
- bootcamp-brian commented on: https://github.com/hackforla/website/pull/4600#issuecomment-1544789849 at 05/11/2023, 03:49 PM PDT
- bootcamp-brian unassigned: https://github.com/hackforla/website/issues/4605 at 05/13/2023, 08:53 PM PDT
- bootcamp-brian PR was merged: https://github.com/hackforla/website/pull/4600#event-9283569419 at 05/18/2023, 01:33 PM PDT
- bootcamp-brian issue closed by PR 4600: https://github.com/hackforla/website/issues/4249#event-9283569630 at 05/18/2023, 01:33 PM PDT
- bootcamp-brian merged: https://github.com/hackforla/website/pull/4691#event-9286175614 at 05/18/2023, 11:30 PM PDT
- bootcamp-brian closed by PR 4691: https://github.com/hackforla/website/issues/4687#event-9286175735 at 05/18/2023, 11:30 PM PDT
